assets/src/edit-story/karma/fixture.js

```diff
@@ -303,9 +307,10 @@ class ComponentStub {
       );
 
       return (
-        <HookExecutor key={refresher} hooks={hooks}>
+        <>
+          <HookExecutor key={refresher} hooks={hooks} />
```

Comment:
Ah great - I actually ran into this many times and thought it was unavoidable, but awesome that you were able to fix it!